### PR TITLE
[login] Test instructions

### LINKED ISSUE #6960: [login] Test instructions
In the request account test plan: 
- Instruction 6: instead of user_accounts and user_accounts_multisite use the name on the front end (User Management and Across all sites create and edit users) or mention it in parentheses
- Instruction 12: typo at the end of instruction

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Link(s) to related issue(s)

* Resolves #6960  (Reference the issue this fixes, if any.)

### CODE REVIEW

File: modules/login/test/Request_Account_test_plan.md [hunk 1]
```diff
@@ -17,6 +17,5 @@
 ### Email functionality
 11. Disable the `sandbox` configuration setting in your configuration file.
 12. Create a new user account using an email address you can access. Give it either the `user_accounts` or `user_accounts_multisite` permission.
-permission and has
 12. Issue a new user account request specifying the same site and project as the user you created in the previous step. Verify that an email is sent to your email address.
```
Comment (laemtl) [inline modules/login/test/Request_Account_test_plan.md:20]: This should be 13.

**Review by laemtl — CHANGES_REQUESTED** [review]

File: modules/login/test/Request_Account_test_plan.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 5. Verify "Return to Loris login page" link on Thank you page works
 	
 ### Approving new User Account Request:
-6. Log in as another user who has permission:user_accounts and does Not have permission:user_accounts_multisite.  Verify that new account request notification is counted in Dashboard (count has incremented).
+6. Log in as another user who has permission:user_accounts and does Not have permission:user_accounts_multisite (User Management and Across all sites create and edit users).  Verify that new account request notification is counted in Dashboard (count has incremented).
```
Comment (laemtl) [inline modules/login/test/Request_Account_test_plan.md:11]: - remove the double space before `Verify that new account`
- add space after `permission:` for clarity

**Review by laemtl — CHANGES_REQUESTED** [review]

File: modules/login/test/Request_Account_test_plan.md (hunk 2 above)
Comment (haoweiqiu) [reply modules/login/test/Request_Account_test_plan.md]: Hi, what do you mean by the last point.

Comment (laemtl) [reply modules/login/test/Request_Account_test_plan.md]: 6. Log in as another user who has permission `user_accounts` _(insert frontend lable here)_ and does not have permission `user_accounts_multisite` (User Management and Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).

Comment (haoweiqiu) [reply modules/login/test/Request_Account_test_plan.md]: Alright, thanks.

File: modules/login/test/Request_Account_test_plan.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 5. Verify "Return to Loris login page" link on Thank you page works
 	
 ### Approving new User Account Request:
-6. Log in as another user who has permission: user_accounts and does not have permission:user_accounts_multisite (User Management and Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).
+6. Log in as another user who has permission: user_accounts (User Management) and does not have permission:user_accounts_multisite (Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).
```
Comment (laemtl) [inline modules/login/test/Request_Account_test_plan.md:11]: 6. Log in as another user who has permission: `user_accounts` (User Management) and does not have permission: `user_accounts_multisite` (Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).

**Review by laemtl — CHANGES_REQUESTED** [review]
You can replace the content of point 6 with my comment. I added quotes around the permission name (a convention I noticed in a lot of test plan) and a missing space after a semi-colon. Thanks!

File: modules/login/test/Request_Account_test_plan.md (hunk 3 above)
Comment (haoweiqiu) [reply modules/login/test/Request_Account_test_plan.md:11]: Oops, sorry..

**Review by laemtl — APPROVED** [review]

**Review by maltheism — APPROVED** [review]

**Pull request merged**